test/e2e/test.utils.js

```diff
@@ -10,3 +10,16 @@ export const accountAddresses = {
   gov1: 'agoric1ee9hr0jyrxhy999y755mp862ljgycmwyp4pl7q',
   gov2: 'agoric1wrfh296eu2z34p6pah7q04jjuyj3mxu9v98277',
 };
+
+export const MINUTE_MS = 60000;
+
```

Comment:
`export const DEFAULT_TIMEOUT = 1 * 60 * 1000;` is more semantic. 
Given we're using `MINUTE_MS` at multiple places. Perhaps it should be called `DEFAULT_TIMEOUT`?

Reply:
i think the name of this variable gives a clear enough indication of the number it represents. it can be left as is